yt/docs/en/_includes/admin-guide/identity-sync.md

```diff
@@ -0,0 +1,17 @@
+# Import users and groups from the external systems.
+
+Manual management {{product-name}}' users and groups is discussed in the section [Managing users, groups](../../admin-guide/cluster-operations.md#managing-users,-groups-and-access-controls).  
+Sometimes it is required to keep users and group lists in sync with an external system.
```

Comment:
1. I would recommend to separate this sentence from the previous paragraph with a blank line and to combine it with the next sentence:
```
{{product-name}}' users and groups manual management is discussed in the section [Managing users, groups](../../admin-guide/cluster-operations.md#managing-users,-groups-and-access-controls).  

Sometimes it is required to keep users and group lists in sync with an external system. There is an app [ytsaurus- 
identity-sync](https://github.com/tractoai/ytsaurus-identity-sync) which might help when such an issue arises.
```
2. There is an extra line break before a word `which`.